docs/cn/ufs/Minio.md

```diff
@@ -48,5 +49,98 @@ aws.accessKeyId=<MINIO_ACCESS_KEY_ID>
+
+
+```
+Exception in thread "main" alluxio.exception.DirectoryNotEmptyException: Failed to delete /default_tests_files (com.amazonaws.SdkClientException: Unable to execute HTTP request: {{BUCKET_NAME}}) from the under file system
+        at alluxio.client.file.BaseFileSystem.delete(BaseFileSystem.java:234)
+        at alluxio.cli.TestRunner.runTests(TestRunner.java:118)
+        at alluxio.cli.TestRunner.main(TestRunner.java:100)
+```
+###连接被拒绝-无法执行HTTP请求
+
+如果发生客户端返回连接被拒绝错误的异常，
+那么很可能是Alluxio无法连接MinIO服务器。
+确保
+[`alluxio.underfs.s3.endpoint`]({{ '/en/reference/Properties-List.html' | relativize_url}}#alluxio.underfs.s3.endpoint)
+设值是正确的，并且运行Alluxio的master节点可以通过网络连接到MinIO endpoint。
+
+```
+Exception in thread "main" alluxio.exception.DirectoryNotEmptyException: Failed to delete /default_tests_files (com.amazonaws.SdkClientException: Unable to execute HTTP request: Connect to localhost:9001 [localhost/127.0.0.1] failed: Connection refused (Connect
+ion refused)) from the under file system
+        at alluxio.client.file.BaseFileSystem.delete(BaseFileSystem.java:234)
+        at alluxio.cli.TestRunner.runTests(TestRunner.java:118)
+        at alluxio.cli.TestRunner.main(TestRunner.java:100)
+```
+
+###请求禁止
+
+如果遇到包含有关禁止访问的消息的异常，则可能
+Alluxio master凭证配置不正确。
+检查[`aws.accessKeyId`]({{ '/en/reference/Properties-List.html' | relativize_url}}#aws.accessKeyId)
+和[`aws.secretKey`]({{ '/en/reference/Properties-List.html' | relativize_url}}#aws.secretKey)。
+如果出现此错误，请仔细检查这两个属性的设置是否正确。
+
+
+```
+ERROR CliUtils - Exception running test: alluxio.examples.BasicNonByteBufferOperations@388526fb
+alluxio.exception.AlluxioException: com.amazonaws.services.s3.model.AmazonS3Exception: Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 1586BF770688AB20; S3 Extended Request ID: null), S3 Extended Request ID: null
+        at alluxio.exception.status.AlluxioStatusException.toAlluxioException(AlluxioStatusException.java:111)
+        at alluxio.client.file.BaseFileSystem.createFile(BaseFileSystem.java:200)
+        at alluxio.examples.BasicNonByteBufferOperations.createFile(BasicNonByteBufferOperations.java:102)
+        at alluxio.examples.BasicNonByteBufferOperations.write(BasicNonByteBufferOperations.java:85)
+        at alluxio.examples.BasicNonByteBufferOperations.call(BasicNonByteBufferOperations.java:80)
+        at alluxio.examples.BasicNonByteBufferOperations.call(BasicNonByteBufferOperations.java:49)
+        at alluxio.cli.CliUtils.runExample(CliUtils.java:51)
+        at alluxio.cli.TestRunner.runTest(TestRunner.java:164)
+        at alluxio.cli.TestRunner.runTests(TestRunner.java:134)
+        at alluxio.cli.TestRunner.main(TestRunner.java:100)
+Caused by: alluxio.exception.status.UnknownException: com.amazonaws.services.s3.model.AmazonS3Exception: Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 1586BF770688AB20; S3 Extended Request ID: null), S3 Extended Request ID: null
+```
```

Comment:
There is a code convention to always add an empty line at the end of the file. Github website typically doesn't show that last empty line so it's hard for review. I checkout this PR locally, and found there is no empty line at the end of file. Please add one empty line at the end of the file